src/erlcloud_aws.erl

```diff
@@ -336,13 +336,30 @@ default_config_env() ->
 %% return <code>undefined</code>.
 %%
 auto_config() ->
+    auto_config( [] ).
+
+
+%%%---------------------------------------------------------------------------
+-spec auto_config( ProfileOptions :: [profile_option()]) ->
+                         {ok, aws_config()} | undefined.
+%%%---------------------------------------------------------------------------
+%% @doc Generate config using the best available credentials
+%%
+%% This function works the same as {@link auto_config/0}, but if credentials
+%% are developed from <em>User Profile</code> as the source, the
```

Comment:
i mean, i could just not write doc like the rest of this code, and then i wouldn't have this problem...